Classes/Domain/Model/AbstractObject.php

```diff
@@ -466,15 +468,21 @@ public function setDocComment($docComment)
             if ($line !== '' && strpos($line, '* @') !== false) {
                 $this->parseTag(substr($line, strpos($line, '@')));
             } else {
-                if (count($this->tags) === 0) {
-                    $this->description .= preg_replace('/\\s*\\/?[\\\\*]*\\s?(.*)$/', '$1', $line) . PHP_EOL;
+                $plainLine = preg_replace('/\\s*\\/?[\\\\*]*\\s?(.*)$/', '$1', $line);
```

Comment:
`preg_replace('#\\s*/?[\\\\*]*\\s?(.*)$#', '$1', $line);`
is IMO more readable.
Shouldn't we anchor it on `^` otherwise this would be captured too: 
`foo /* the phpdoc`

`[\\\\*]` why allowing a combination of backslashes and asterisks?

Reply:
I just moved the existing regex without refactoring it. You may change it but it's not part of this change I would say...

Reply:
Oh sorry, I did not see that.